front/components/assistant/conversation/InputBar.tsx

```diff
@@ -753,13 +760,54 @@ export function FixedAssistantInputBar({
   owner,
   onSubmit,
   stickyMentions,
+  conversationId,
 }: {
   owner: WorkspaceType;
   onSubmit: (input: string, mentions: MentionType[]) => void;
   stickyMentions?: AgentMention[];
+  conversationId: string | null;
 }) {
+  // GenerationContext: to know if we are generating or not
+  const generationContext = useContext(GenerationContext);
+  if (!generationContext) {
+    throw new Error(
+      "FixedAssistantInputBar must be used within a GenerationContextProvider"
```

Comment:
We're not in FixedAssistantInputBar here?

Reply:
This time we are! I got at least one with the valid name 😄